deploy/rbac-clowdapp.yml

```diff
@@ -0,0 +1,555 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: rbac
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: rbac
+  spec:
+    envName: ${ENV_NAME}
+    pods:
```

Comment:
We should move this to newer deployments style

Reply:
done!